packages/@aws-cdk/aws-events/lib/target.ts

```diff
@@ -62,4 +63,6 @@ export interface RuleTargetConfig {
    * @default the entire event
    */
   readonly input?: RuleTargetInput;
+
+  readonly target?: IConstruct;
```

Comment:
Documentation

Reply:
I think something like `targetResource`. Please also provide the value for the other EventTargets.

Reply:
Done, done and done.